usaspending_api/api_contracts/contracts/v2/agency/toptier_code/def_code.md

```diff
@@ -0,0 +1,106 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# List Disaster Emergency Fund Codes [/api/v2/agency/{toptier_code}/def_code/{?fiscal_year,filter,order,sort,page,limit}]
+
+Returns a list of Disaster Emergency Fund Codes in the Agency's appropriations for a single fiscal year
+
+## GET
+
++ Request (application/json)
```

Comment:
I believe lines 10-16 are unnecessary for GET requests, since the request is not in JSON format (the mock API doesn't work for me unless I remove those lines).